compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/SymbolFinder.java

```diff
@@ -1264,6 +1264,10 @@ public void visit(BLangTupleTypeNode tupleTypeNode) {
     @Override
     public void visit(BLangErrorType errorType) {
         lookupNode(errorType.detailType);
+
+        if (symbolAtCursor == null && errorType.getBType() != null) {
```

Comment:
Was there a case where the type gets null?

Reply:
This was an additional check placed in order to avoid a possible NPE. Removed it since no instances were found where the type gets null.